CHANGELOG.md

```diff
@@ -1,317 +1,853 @@
- * [[`b6357cc323`](https://github.com/level/levelup/commit/b6357cc323)] - Use highest/lowest instead of largest/smallest (Arnout Engelen)
- * [[`e45ce4e9d5`](https://github.com/level/levelup/commit/e45ce4e9d5)] - Document what 'limit' does in 'reverse' mode (Arnout Engelen)
- * [[`9004e9db69`](https://github.com/level/levelup/commit/9004e9db69)] - Discourage the use of start/end a bit (Arnout Engelen)
- * [[`6178d69cdc`](https://github.com/level/levelup/commit/6178d69cdc)] - merge (Dominic Tarr)
- * [[`e3ab0ebe03`](https://github.com/level/levelup/commit/e3ab0ebe03)] - document with comments (Dominic Tarr)
- * [[`bb88572c15`](https://github.com/level/levelup/commit/bb88572c15)] - unmention bops (Dominic Tarr)
- * [[`45df8e668e`](https://github.com/level/levelup/commit/45df8e668e)] - clarify ltgt (Dominic Tarr)
- * [[`54eba03305`](https://github.com/level/levelup/commit/54eba03305)] - binary encoding in the browser (Calvin Metcalf)
-
-## 0.18.6 @ Jul 26 2014
-
- * (lots of stuff since 0.18.1 not recorded in changelog, ooops)
- * Extracted encoding / codec to separate modules (@dominictarr)
-
-## 0.18.1 @ Nov 20 2013
-
- * Make chained-batch obey global LevelUP object options (@mcavage)
-
-## 0.18.0 @ Nov 18 2013
-
- * Upgrade to LevelDOWN@0.10.0 (and bops@0.1.0 and readable-stream@1.1.9) (@rvagg)
-
-## 0.17.0 @ Oct 01 2013
-
- * Undo factory pattern, use plain prototypal object and expose full prototype (@rvagg)
- * Move Batch object to batch.js and expose (@rvagg)
- * Use new package, DeferredLevelDOWN to handle all deferred open logic (@rvagg)
- * Code cleanup, update deps (xtend) (@rvagg, @juliangruber)
-
-## 0.16.0 @ Sep 10 2013
-
- * Added `notFound` boolean property and `status=404` property to NotFoundError (@rvagg)
- * Upgrade to errno@0.1.0 which aliases .type and .name properties (@rvagg)
- * ReadStream gracefully handles multiple destroy() calls (@mcollina)
-
-## 0.15.0 @ Aug 25 2013
-
- * New ReadStream: upgrade to streams2, remove all state-management cruft, remove fstream support (@substack)
- * Update LevelDOWN dependency to ~0.8.0 with Iterator lt/lte/gt/gte support and NAN as a dependency
- * Added @substack as contributor
-
-## 0.14.0 @ Aug 19 2013
-
- * Encodings overhaul, allow custom encoders/decoders for `keyEncoding` or `valueEncoding` (@dominictarr)
-
-## 0.13.0 @ Aug 11 2013
-
- * Update LevelDOWN dependency version ~0.7.0 for Node 0.8->0.11 compatibility
-
-## 0.12.0 @ Jul 25 2013
-
-  * Update LevelDOWN dependency version ~0.6.2
-
-## 0.11.0 @ Jul 17 2013
-
-  * Remove all Function#bind calls for better browser compatibility (@juliangruber)
-  * Switch from direct Buffer access to bops for better browser compatibility (@juliangruber)
-  * WriteStream#end accepts `data` argument (@pgte)
-  * Added @pgte as contributor
+# Changelog
+
+## [Unreleased]
+
+### Added
+* Add 9 to Travis (@ralphtheninja)
+
+### Changed
+* Update `browserify` (@ralphtheninja)
+* Update `leveldown` (@ralphtheninja)
```

Comment:
If these are major updates, maybe we should mention the version (same for similar entries below).

Reply:
Updating. Maybe in the future we should actually write the version for all dependencies, even if it's only a minor or a patch.